### DISCUSSION TITLE
Welcome to OctoPrint-DiscordRemote Discussions!

### DISCUSSION THREAD

**Discussion by cameroncros** [cameroncros/OctoPrint-DiscordRemote#166] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TechGoddessKat** [comment]
I was curious if/when support for the bot responding to commands in multiple channels/servers may be implemented?

**Reply by cameroncros** [reply]
At the moment, no plans, unless someone works on it. Perhaps once the discord.py transition occurs, it might be easier?

**Reply by TechGoddessKat** [reply]
Thank you for answering! It is unfortunate that the feature isn't currently in development but that is understandable with such as this project is rather small. I see great promise in it though and I hope to see it grow even larger as it is such a useful and helpful project. Thank you again for answering me and thank you for creating this amazing project!

**Comment by JadePrincess89** [comment]
so... Octolapse has a gcode trigger snapshot command  " @OCTOLAPSE TAKE-SNAPSHOT " that i can put into the gcode... does discord remote have something like that?

or maybe have a snapshot at layer change...

**Comment by cameroncros** [comment]
No, at the moment it doesn't have something like that.

It may be a relatively small change though? Could probably copy whatever
Octolapse did to detect the gcode and just have it work.

I am not actively developing Discord remote, but I am happy to guide anyone
who wants to make changes.
